serverless.yml

```diff
@@ -1,32 +1,29 @@
 # serverless.yml
 
-service: IOW-RETRIEVER-CAPTURE
+service: aqts-retriever-capture-raw-load
 
 provider:
   name: aws
   region: 'us-west-2'
   stage: ${opt:stage, 'TEST'}
-  runtime: python3.7
+  runtime: python3.8
   memorySize: 256
   logRetentionInDays: 30
   deploymentBucket:
     name: iow-cloud-applications
   stackTags:
-    "wma:applicationId": "Retriever Capture Loader"
+    "wma:applicationId": "Aquarius TS Retriever Capture Raw Loader"
     "wma:contact": "Jim Kreft jkreft@usgs.gov"
     "wma:environment": ${self:provider.stage}
 
 custom:
   accountNumber : ${ssm:/iow/aws/accountNumber}
   vpc:
-    securityGroupIds:
-      - ${ssm:/iow/retriever-capture/vpc/securityGroupId}
-    subnetIds:
-      - ${ssm:/iow/retriever-capture/vpc/subnetId}
+    securityGroupIds: ${ssm:/iow/retriever-capture/${self:provider.stage}/securityGroupIds~split}
+    subnetIds: ${ssm:/iow/aws/vpc/${self:provider.stage}/subnetIds~split}
```

Comment:
I've modified the convention from Ed's example -- subnets are defined within the VPC and are stored as a list in Parameter Store. Security groups are also stored as a list, but I opted to keep them application specific because there may be applications that require more specialized security groups.